app/modules/ph/cruds_ph.py

```diff
@@ -0,0 +1,69 @@
+from collections.abc import Sequence
+from datetime import date
+
+from sqlalchemy import delete, select, update
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.modules.ph import models_ph, schemas_ph
+
+
+async def get_papers(
+    db: AsyncSession,
+    end_date: date = date(2099, 12, 31),
+) -> Sequence[models_ph.Paper]:
+    """Return papers from the latest to the oldest"""
+    result = await db.execute(
+        select(models_ph.Paper)
+        .where(models_ph.Paper.release_date <= end_date)
+        .order_by(models_ph.Paper.release_date.desc()),
+    )
+    return result.scalars().all()
```

Comment:
I'm not completely fond of this magic value `31/12/2099`.
Maybe you could do:
```suggestion
    end_date: date | None = None,
) -> Sequence[models_ph.Paper]:
    """Return papers from the latest to the oldest"""
    result = await db.execute(
        select(models_ph.Paper)
        .where(models_ph.Paper.release_date <= (end_date or date.max))
        .order_by(models_ph.Paper.release_date.desc()),
    )
    return result.scalars().all()
```